underfs/cephfs/src/main/java/alluxio/underfs/cephfs/CephInputStream.java

```diff
@@ -0,0 +1,131 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.underfs.cephfs;
+
+import com.ceph.fs.CephMount;
+
+import javax.annotation.concurrent.NotThreadSafe;
+
+import java.io.InputStream;
+import java.io.IOException;
+
+import java.util.concurrent.atomic.AtomicBoolean;
+
+/**
+ * A stream for reading a file from CephFS.
+ */
+@NotThreadSafe
+public final class CephInputStream extends InputStream {
+  private CephMount mMount;
+  private int mFileHandle;
```

Comment:
mark `final` whenever  possible
```suggestion
  private final int mFileHandle;
```